modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -94,7 +94,10 @@ $ mkdir $HOME/clusterconfig
 [source,terminal]
 ----
 $ openshift-install create manifests --dir $HOME/clusterconfig
-? SSH Public Key ...
+INFO Credentials loaded from the "myprofile" profile in file "/home/myuser/.aws/credentials"
+INFO Consuming Install Config from target directory
+INFO Manifests created in: $HOME/clusterconfig/manifests and $HOME/clusterconfig/openshift
+
```

Comment:
Based on the [code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) section of the docs guidelines, I'm requesting a few changes:
* Please create a separate code block for the example output that you added.
* Because this code block transgressed the guideline about using a single command--the `ls` command is verification--please insert an additional step.  I believe that the instructional text can be "Optional: Confirm that the installation program created manifests in the `clusterconfig/openshift` directory:"
* Also add a separate code block for the example output of the `ls` command.

Reply:
Acked. Included the changes.

Reply:
@mikemckiernan Does this look good to merge? or Does it need any change?

Reply:
Apologies, I wasn't clear.  The doc guidelines indicate to use one code block for the command and a separate code block for the command output.  Can you revise like the foll?

```
. Run `openshift-install` to create a set of files in the `manifest` and `openshift` subdirectories. Answer the system questions as you are prompted:
+
[source,terminal]
----
$ openshift-install create manifests --dir $HOME/clusterconfig
---
+
.Example output
+
[source,terminal]
----
? SSH Public Key ...
INFO Credentials loaded from the "myprofile" profile in file "/home/myuser/.aws/credentials"
INFO Consuming Install Config from target directory
INFO Manifests created in: $HOME/clusterconfig/manifests and $HOME/clusterconfig/openshift
----

. Optional: Confirm that the installation program created manifests in the `clusterconfig/openshift` directory:
+
[source,terminal]
----
$ ls $HOME/clusterconfig/openshift/
----
+
.Example output
+
[source,terminal]
----
99_kubeadmin-password-secret.yaml
99_openshift-cluster-api_master-machines-0.yaml
99_openshift-cluster-api_master-machines-1.yaml
99_openshift-cluster-api_master-machines-2.yaml
...
----
```

It's definitely verbose, but those are the guidelines.